script/Arcueid/Day 11/11_00_ARC11_2.txt

```diff
@@ -377,7 +377,7 @@ A disposition so sunny it's bordering on airheaded,
 [sha:a0acd44f024f533b7d766ec809f00e78b52cdc23]{
 -- Page 63, Offset 29678.
 -- 真祖と呼ばれる吸血種には、死徒たちよりはるかに業の深い吸血衝動がある。
-The vampires we call True Ancestors possess a bloodthirsty urge far more sinful than that of the Dead Apostles.
+The vampires we call True Ancestors possess vampiric impulses far more sinful than those of the Dead Apostles.
```

Comment:
found 吸血衝動 translated in most instances as a general term, in which I think these impulses are a special type of urges?

Reply:
I had been using vampiric impulse because it sounds nicer than "bloodsucking impulse/urge". It also works linguistically because 吸血 is the shared prefix

Reply:
吸血衝動 is a term, and always refers to the peculiar form of 'hunger' for blood felt by the True Ancestors.

In the past it was always translated as "Bloodsucking Impulse" but I think "vampiric urges" (non-capitalized) sounds much better. If you wanna make that change though it has to be consistent across the entire project, which means you should bring it up in the terms channel.

Reply:
Another thing: 吸血種 should be differentiated from 吸血鬼. In fact, this whole paradigm might be why 吸血衝動 was called "bloodsucking impulse" in the first place. 

吸血種 is usually translated as bloodsucker, while 吸血鬼 is vampire. This needs to be consistent because there are 吸血種 who are not 吸血鬼.